packages/dkml-runtime-distribution/dkml-runtime-distribution.1.0.2~prerel7/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+synopsis: "Scripts used by the Diskuv OCaml distribution during installation"
+description: """\
+Scripts used by the Diskuv OCaml distribution during the installation of:
+* a local project (ie. a Opam switch created with `opam dkml init`)
+* a user profile (ex. OCaml binaries installed within the user's home directory)
+* a machine (ex. system or Administrator assembly/C compilers)"""
+maintainer: "opensource+diskuv-ocaml@support.diskuv.com"
+authors: "Diskuv, Inc. <opensource+diskuv-ocaml@support.diskuv.com>"
+license: "Apache-2.0"
+homepage: "https://github.com/diskuv/dkml-runtime-distribution"
+bug-reports: "https://github.com/diskuv/dkml-runtime-distribution/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ocaml" {>= "4.12.1~" & < "4.12.2~" | >= "4.13.1~" & < "4.13.2~"}
```

Comment:
Is there a reason why the constraint here is so strict?

Reply:
This package is the central meta package for DKML.  https://github.com/diskuv/dkml-runtime-distribution/tree/main/src/none is made into a library, and today DKML only supports 4.12.1 and 4.13.1. The library is used to drive GitLab CI/CD integration testing (analogous to Opam CI) among other things. When I get time and CI/CD resources that constraint will loosen.